src/sentry/metrics/dogstatsd.py

```diff
@@ -1,45 +1,40 @@
 import atexit
 from typing import Any
 
-from datadog import initialize
-from datadog.dogstatsd.base import statsd
+from datadog.dogstatsd.base import DogStatsd
 
 from .base import MetricsBackend, Tags
 
 __all__ = ["DogStatsdMetricsBackend"]
 
-# Set the maximum number of packets to queue for the sender.
-# How may packets to queue before blocking or dropping the packet if the packet queue is already full.
-# 0 means unlimited.
-SENDER_QUEUE_SIZE = 0
-
-# Set timeout for packet queue operations, in seconds
-# How long the application thread is willing to wait for the queue clear up before dropping the metric packet.
-# If set to None, wait forever.
-# If set to zero drop the packet immediately if the queue is full.
-SENDER_QUEUE_TIMEOUT = 0
-
 
 class DogStatsdMetricsBackend(MetricsBackend):
     def __init__(self, prefix: str | None = None, **kwargs: Any) -> None:
-        # TODO(dcramer): it'd be nice if the initialize call wasn't a global
         self.tags = kwargs.pop("tags", None)
-        kwargs["statsd_disable_buffering"] = False
-
-        initialize(**kwargs)
-        statsd.disable_telemetry()
 
-        # When enabled, a background thread will be used to send metric payloads to the Agent.
-        statsd.enable_background_sender(
-            sender_queue_size=SENDER_QUEUE_SIZE, sender_queue_timeout=SENDER_QUEUE_TIMEOUT
-        )
-        # Applications should call wait_for_pending() before exiting to make sure all pending payloads are sent.
-        atexit.register(statsd.wait_for_pending)
+        instance_kwargs: dict[str, Any] = {
+            "disable_telemetry": True,
+            "disable_buffering": False,
+            "disable_background_sender": False,
```

Comment:
Let's keep the comment on this

```suggestion
            # When enabled, a background thread will be used to send metric payloads to the Agent.
            "disable_background_sender": False,
```